dll/win32/kernel32/client/thread.c

```diff
@@ -1251,4 +1251,194 @@ TlsSetValue(IN DWORD Index,
     return TRUE;
 }
 
+
```

Comment:
I think putting this stuff in a (new) file under dll/win32/kernel32/wine might make more sense for easier syncing. Also please add a mention in README.WINE.